### PR TITLE
Fix "Avoid mutable default arguments" issue

### PR DESCRIPTION
**Pull request by orbeckst**
This pull request automatically fixes all occurrences of the following issue:

Issue type: [Avoid mutable default arguments](https://www.quantifiedcode.com/app/issue_class/3P0qV6OB)
Issue details: [https://www.quantifiedcode.com/app/project/gh:MDAnalysis:mdanalysis?groups=code_patterns/%3A3P0qV6OB](https://www.quantifiedcode.com/app/project/gh:MDAnalysis:mdanalysis?groups=code_patterns/%3A3P0qV6OB)

To adjust the commit message or the actual code changes, just [rebase](https://git-scm.com/docs/git-rebase) or [cherry-pick](https://git-scm.com/docs/git-cherry-pick) the commits.
For questions or feedback reach out to cody@quantifiedcode.com.

Legal note: We won't claim any copyrights on the code changes.

Cheers,
[Cody - Your code quality bot](https://www.quantifiedcode.com/cody)

### CODE REVIEW

File: package/MDAnalysis/coordinates/DMS.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ def get_coordinates(self, cur):
         particles = cur.fetchall()
         return [(p['x'], p['y'], p['z']) for p in particles]
 
-    def get_particle_by_columns(self, cur, columns=['x', 'y', 'z']):
+    def get_particle_by_columns(self, cur, columns=None):
```
Comment (orbeckst) [inline package/MDAnalysis/coordinates/DMS.py]: Good work, @quantifiedcode-bot  :-)

Comment (mnmelo) [inline package/MDAnalysis/coordinates/DMS.py]: Ah, I'm on holidays and haven't had time to carefully look at these. However, I'll disagree with @orbeckst that Cody did a good job here. It'd be cleaner and more concise to replace the list with a tuple (which is immutable and fulfills the task just as well).

Comment (programmdesign) [inline package/MDAnalysis/coordinates/DMS.py]: @mnmelo Thanks for the feedback. We don't change your arguments with this rewrite as changing too much code would increase the risk of breaking your code. Replacing a list with a tuple might not always be safe.

Comment (mnmelo) [inline package/MDAnalysis/coordinates/DMS.py]: @programmdesign, I totally understand that this is the conservative approach. Making it cleverer in the sense I mentioned is probably not even possible within reasonable effort. Just stating that this is one of the cases where we might want to not follow Cody's suggestion.

Comment (programmdesign) [inline package/MDAnalysis/coordinates/DMS.py]: @mnmelo The mutable default argument check only improves the handling of the mutable arguments, not the arguments themselves. As you said, that is a lot more effort. So Cody "suggests" only, what's in your code already ;-)

**Pull request merged**